clients/banking/src/components/TransferInternationalWizardBeneficiary.tsx

```diff
@@ -147,16 +160,37 @@ export const TransferInternationalWizardBeneficiary = ({
     listenFields(["results"], () => refresh());
   }, [listenFields, refresh]);
```

Comment:
Don't forget to remove the listeners to avoid leaks / updating unmounted components states:

```ts
useEffect(() => {
  const removeRouteListener = listenFields(["route"], ({ route: { value } }) => setRoute(value));
  const removeResultsListener = listenFields(["results"], () => refresh());

  return () => {
    removeRouteListener();
    removeResultsListener();
  };
}, [listenFields, refresh]);
```